### PR TITLE
fix sample app's build issues

### PR DESCRIPTION
**Pull request by thuyBox**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @thuyBox has signed the CLA. Thanks for the pull request!

File: BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.h [hunk 1]
```diff
@@ -7,11 +7,9 @@
 //
 
 #import <Foundation/Foundation.h>
-
-#import "BOXOAuth2Session.h"
-
 #import "BOXContentSDKConstants.h"
 #import "BOXURLSessionManager.h"
+#import "BOXAbstractSession.h"
```
Comment (boxtcarpel) [inline BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.h]: Can just forward declare the class, no? (actual import can be in the .m).

Comment (boxtcarpel) [reply BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.h]: Which is covered by the subclass 'BOXOAuth2Session' which is already imported.

**Review by boxtcarpel — APPROVED** [review]

File: BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.h (hunk 1 above)
Comment (thuyBox) [reply BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.h]: done

**Review by boxtcarpel — APPROVED** [review]

**Pull request merged**